slicer.py

```diff
@@ -131,9 +131,11 @@ def parse_currency_data(filename):
                 if localization.lower() in fields[0].lower():
                     currency = 'USD - RoW'
  
-        exchange_rate = Decimal(fields[column_index_exchange_rate].replace(',', ''))
         amount_pre_tax = Decimal(fields[column_index_amount_pre_tax].replace(',', ''))
         amount_after_tax = Decimal(fields[column_index_amount_after_tax].replace(',', ''))
+        earnings = Decimal(fields[column_index_earnings].replace(',', ''))
```

Comment:
Please add a comment explaining why we explicitly calculate the exchange rate instead of using the one provided by Apple.
```suggestion
        # calculate the exchange rate explicitly instead of relying on the "Exchange Rate“ column
        # because its value is rounded to 6 decimal places and sometimes not precise enough
        earnings = Decimal(fields[column_index_earnings].replace(',', ''))
```